### PR TITLE
Prevent flash

### PR DESCRIPTION
**Pull request by indirectlylit**
addresses

* https://trello.com/c/ZLqC5fLw/591-import-export-minor-flash-within-modal-upon-clicking-export
* https://trello.com/c/0FgpjK7q/594-import-export-refresh-ux

additional drive-by change: remove unnecessary focus outline on modals

### CODE REVIEW

File: kolibri/core/assets/src/vue/core-modal/index.vue [hunk 1]
```diff
@@ -154,6 +154,10 @@
     transition: all 0.3s ease
     margin: 0 auto
     padding: 15px 30px
+
+    &:focus
+      outline: none
+
```
Comment (radinamatic) [inline kolibri/core/assets/src/vue/core-modal/index.vue:160]: Oh, I love when all of you take stuff off my a11y-todo notes! 
:+1: :heart:

File: kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-export.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@
           </icon-button>
         </div>
       </template>
-      <loading-spinner v-else :delay="0" class="spinner"></loading-spinner>
+      <loading-spinner v-else :delay="500" class="spinner"></loading-spinner>
```
Comment (christianmemije) [inline kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-export.vue:43]: Just noticed you had added this delay data binding, seems really helpful! :+1:

**Comment by indirectlylit** [comment]
@christianmemije mind reviewing again and approving if appropriate?

I added a new commit to prevent the 'cancel' button from being disabled when the modal is refreshing: https://github.com/learningequality/kolibri/pull/545/commits/660bd66bbceb474425806a93589fd6cbb7c35be3

**Comment by christianmemije** [comment]
@indirectlylit Tests are failing

**Comment by indirectlylit** [comment]
looks unrelated to this PR - they all say `ERROR: unknown environment 'lint'`

**Comment by indirectlylit** [comment]
rebased with master and re-pushed. @ralphiee22 says we can check travis for cache issues if that doesn't work.

**Pull request merged**